tools/unrmdp.cpp

```diff
@@ -43,25 +46,30 @@ int main(int argc, char** argv) {
 	app.add_option("rmdpfile", rmdpFile, "The rmdp file containing the archives raw data")
 			->check(CLI::ExistingFile)
 			->required();
-	
+
 	app.add_flag("-l, --list", onlyListFiles, "List files in an archive without actually extracting them");
 
 	CLI11_PARSE(app, argc, argv);
 
+	fmt::print("Processing index...\n");
+
 	AWE::RMDPArchive rmdp(
 		new Common::ReadFile(binFile),
 		new Common::ReadFile(rmdpFile)
 	);
 
 	for (size_t i = 0; i < rmdp.getNumResources(); ++i) {
 		const std::string path = rmdp.getResourcePath(i);
+		const std::string normalizedPath = AWE::getNormalizedPath(path);
 
-		fmt::print("{}/{} {}\n", i + 1, rmdp.getNumResources(), path);
+		fmt::print("{}/{} {}\n", i + 1, rmdp.getNumResources(), normalizedPath);
 
 		if (!onlyListFiles) {
-			const auto resourceStream = std::unique_ptr<Common::ReadStream>(rmdp.getResource(path));
+			const auto resourceStream = std::unique_ptr<Common::ReadStream>(rmdp.getResource(Common::toLower(path)));
+			if (!resourceStream.get())
```

Comment:
if (!resourceStream) would be enough here

Reply:
Fixed in the latest commit